modules/AppConfiguration/configurationStores/README.md

```diff
@@ -304,11 +304,7 @@ privateEndpoints:  [
 
 ## Cross-referenced modules
 
-This section gives you an overview of all local-referenced module files (i.e., other CARML modules that are referenced in this module) and all remote-referenced files (i.e., Bicep modules that are referenced from a Bicep Registry or Template Specs).
-
-| Reference | Type |
-| :-- | :-- |
-| `Microsoft.Network/privateEndpoints` | Local reference |
+_None_
```

Comment:
Something about this feels ... 'off' ;) I suspect the `Set-ModuleReadMe` script doesn't pick up on the cross-references anymore. Either a new bug - or something we should fix right away. Thoughts?

Reply:
Naturally, this applies to all other modules with cross-module references too

Reply:
Fixed the utility and regenerated all readmes. Waiting for rerunning a validation pipeline